ArduPlane/ArduPlane.cpp

```diff
@@ -266,7 +266,7 @@ void Plane::one_second_loop()
     // make it possible to change orientation at runtime
     ahrs.update_orientation();
 
-    adsb.set_stall_speed_cm(aparm.airspeed_min);
+    adsb.set_stall_speed_cm(aparm.airspeed_min * 0.8f); // actual stall speed is 80% of this param
```

Comment:
Unrelated change, that I'm assuming wasn't meant to come in here? I don't see any way to establish that the stall speed is actually 80% of our minimum airspeed. It's very common for this to be different.

Reply:
ARSPD_FBW_MIN description:
Minimum airspeed demanded in automatic throttle modes. Should be set to 20% higher than level flight stall speed.

https://github.com/ArduPilot/ardupilot/blob/master/ArduPlane/Parameters.cpp#L353

Reply:
It doesn't actually get used that way, it's more of a minimum then anything else...

Reply:
the aircraft is still expected to fly as expected at that min airspeed, it's wen you get below it that it does drastic things while it still has flight-surface control.

Reply:
I've removed this